packages/itwinui-css/backstop/tests/menu.html

```diff
@@ -1892,6 +1893,400 @@ <h3>Borderless button</h3>
+          <span class="iui-content iui-multilevel-selected">Option 2F, Option 2N</span>
+          <svg-caret-right-small
+            class="iui-icon"
+            aria-hidden="true"
+          ></svg-caret-right-small>
+        </li>
+        <li
+          class="iui-menu-item iui-disabled"
+          role="menuitem"
+          tabindex="0"
+          id="test-multilevel"
+        >
+          <span class="iui-content">Option 3</span>
+          <svg-caret-right-small
+            class="iui-icon"
+            aria-hidden="true"
+          ></svg-caret-right-small>
+        </li>
+      </ul>
+
+      <br />
+
+      <h3 style="padding-bottom: 12px">Submenu</h3>
+
+      <div
+        class="iui-multilevel"
+        role="menuitem"
+        tabindex="0"
+        id="test-multilevel"
+      >
+        <div class="iui-multilevel-header">
+          <svg-caret-left-small
+            class="iui-icon"
+            aria-hidden="true"
+          ></svg-caret-left-small>
+          <span class="iui-content">Option 1</span>
+        </div>
+        <ul
+          class="iui-menu"
+          role="menu"
+        >
+          <li
+            class="iui-menu-item iui-large"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">
+              <div class="iui-menu-label">Option 1A</div>
+              <div class="iui-menu-description">Lorem ipsum</div>
+            </span>
+          </li>
+          <li
+            class="iui-menu-item iui-large iui-active"
+            role="menuitem"
+            aria-checked="true"
+            tabindex="0"
+          >
+            <span class="iui-content">
+              <div class="iui-menu-label">Option 1B</div>
+              <div class="iui-menu-description">Lorem ipsum</div>
+            </span>
+            <svg-checkmark
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-checkmark>
+          </li>
+          <li
+            class="iui-menu-item iui-large"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">
+              <div class="iui-menu-label">Option 1C</div>
+              <div class="iui-menu-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+                proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+            </span>
+          </li>
+        </ul>
+      </div>
+
+      <br />
+
+      <div
+        class="iui-multilevel"
+        role="menuitem"
+        tabindex="0"
+        id="test-multilevel"
```

Comment:
This markup is invalid.
- any element with `role-menuitem` must be a child of an element with `role-menu`
- `id`s must be globally unique

Also, `tabindex` should not be `0` because arrow keys (not <kbd>Tab</kbd> jey) are supposed to be used for keyboard navigation. But looks like that's an issue with the rest of the page too

Reply:
I fixed the `role` (throughout the page) and `id` issues. I left the `tabindex` as-is because it looked like adding arrow-key navigation would require javascript work and I didn't want to add that at this point. If you think it's important, I can make an issue for it.